src/parser/core/modules/CooldownDowntime.js

```diff
@@ -59,7 +59,7 @@ export default class CooldownDowntime extends Module {
 		if (downtime < 0) {
 			downtime = 0
 		}
-		const possibleNumberOfUses = Math.floor(fightlength/(cooldown*1000))
+		const possibleNumberOfUses = Math.ceil(fightlength/(cooldown*1000))
```

Comment:
Nit; Looking at this in hindsight, you can probably get rid of some of the calculations if you factor the downtime out before calculating the usage count.